test/TestServerProjectsLowLevel/dpg-customization/Customization/DPGClient.cs

```diff
@@ -21,21 +21,11 @@ public partial class DPGClient
         /// <exception cref="ArgumentNullException"> <paramref name="mode"/> is null. </exception>
         public virtual async Task<Response<Product>> GetModelValueAsync(string mode, CancellationToken cancellationToken = default)
         {
-            using var scope = ClientDiagnostics.CreateScope("DPGClient.GetModelValue");
-            scope.Start();
-            try
```

Comment:
As per the guidance [here](https://github.com/azure/autorest.csharp/issues/2127#issuecomment-1086127240), If the convenience method name is different, then create outer scope and inner scope will be suppressed as we turned on `suppressNestedClientActivities` flag.

>If the grow-up method diverges much from the protocol method (i.e. if they have different names or code in the convenience method could take much time), do open a scope in the convenience API and turn on suppression of inner scopes.

@annelo-msft please correct me if I'm wrong.

Reply:
That is actually a merge error. Looks like I've picked the wrong file for resolution

Reply:
Fixed!